src/cljs/main/broadfcui/page/method_repo/create_method.cljs

```diff
@@ -138,6 +148,21 @@
                                             #(swap! state assoc :undo-history
                                                     (js->clj (self :call-method "historySize")))))}]
            [:div {:style {:padding "0.25rem"}}]
+           (style/create-form-label "Documentation (optional)")
+           [markdown/MarkdownEditor {:data-test-id "documentation-field"
+                                     :ref "documentation"
```

Comment:
String refs are being deprecated in react:
```cljs
{:ref #(swap! locals assoc :documentation %)}
```
(unless you just don't want to change these lines, which would be ok too)